source/frontend/src/features/mapSideBar/lease/detail/LeaseTabsContainer.tsx

```diff
@@ -69,19 +69,41 @@ export const LeaseTabsContainer: React.FC<ILeaseTabsContainerProps> = ({
     name: 'Checklist',
   });
 
-  tabViews.push({
-    content: (
-      <LeaseTab
-        leasePage={leasePages.get(LeasePageNames.TENANT)}
-        onEdit={() => setContainerState({ activeEditForm: LeasePageNames.TENANT, isEditing: true })}
-        isEditing={isEditing}
-        formikRef={formikRef}
-        onSuccess={onSuccess}
-      />
-    ),
-    key: LeaseFileTabNames.tenant,
-    name: 'Tenant',
-  });
+  if (lease?.paymentReceivableType.id === 'RCVBL') {
```

Comment:
could this be refactored to just change the name of the tabview instead of having these two if statements?